lib/travis/build/script/addons/coverity_scan.rb

```diff
@@ -73,6 +73,7 @@ def build_command
                 env << "COVERITY_SCAN_BUILD_COMMAND_PREPEND=\"#{@config[:build_command_prepend]}\""
                 env << "COVERITY_SCAN_BRANCH_PATTERN=#{@config[:branch_pattern]}"
                 script.cmd "curl -s #{@config[:build_script_url]} | #{env.join(' ')} bash", echo: true
```

Comment:
If all that is wanted is to fail the build when this `curl` command fails, then it is much cleaner to pass `-f` to `curl` and assert this command:

```ruby
script.cmd "curl -sf #{@config[:build_script_url]} | #{env.join(' ')} bash", echo: true, assert: true
```